catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/internal/Splitter.java

```diff
@@ -17,6 +17,7 @@
 import java.io.Serializable;
 import java.util.Map;
 import java.util.stream.Stream;
+import org.codice.ddf.catalog.ui.exceptions.StopSplitterExecutionException;
 
 /** Split a resource into a stream of resources. */
```

Comment:
❗️ We should have the `Experimental` javadoc notice on this interface.

`<b> This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library. </b>`